spec/queries/project/subscriptions_pledged_value_query_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe Project::SubscriptionsPledgedValueQuery do
+  describe ".call" do
+    let(:subscriptions_pledged_value) { Project::SubscriptionsPledgedValueQuery.new(project) }
```

Comment:
I think you don't need this `let`. You can call `described_class.call(project)` in the `subject`.